src/main/java/org/springframework/retry/annotation/Recover.java

```diff
@@ -42,4 +42,11 @@
 @Documented
 public @interface Recover {
 
+	/**
+	 * Name of this function can be used to provide best match for recoverName in
+	 * Retryable. {@link Retryable#recoverName()}
+	 * @return name of this recover function
+	 */
+	String name() default "";
```

Comment:
Why do we need this?

Why just method name in the `@Retryable` attribute is not enough?

Reply:
I agree, better to have one way of providing explicit recovery method name in `@Retryable` without confusing client with additional name in `@Recover`. So I removed additional parameter from `@Recover`.